### DISCUSSION TITLE
Why do I have a weird padding at the top when I'm using a custom app.js and document.js?

### DISCUSSION THREAD

**Discussion by helloitsm3** [vercel/next.js#12828] [Help] [answered]
![image](https://user-images.githubusercontent.com/13825491/81781425-ab1a1600-952a-11ea-81b7-3f99df32fe3b.png)

Here's my code in index.js
```
import React from "react";

export default function Index() {
  return (
    <div style={{ backgroundColor: "red" }}>
      <h1>NextJS</h1>
    </div>
  );
}
```

**Comment by brunocrosier** [comment, marked as answer]
This is probably because ```h1``` tags [come with a margin by default in Chrome ](https://www.w3schools.com/cssref/css_default_values.asp)

You can inspect the h1 element and see it will have some margin

You can use a "CSS Reset" [like this one](https://hankchizljaw.com/wrote/a-modern-css-reset/) in order to reset all these values

[DM me](https://twitter.com/bruno_crosier) if you need more help :)

**Reply by helloitsm3** [reply]
Thanks for replying. I manage to fix that. CSS wasn't resetted even though I refreshed the page. Had to manually hard refresh the page to reset the CSS. Thanks for your help

**Comment by brunocrosier** [comment]
Great - please go ahead and mark this as resolved :)

**Reply by timneutkens** [reply]
Did it for you 🙏 thanks for answering.
